src/stream-ciphers.rst

```diff
@@ -44,29 +44,29 @@ mode, identical input blocks will always map to identical output blocks:
    \overbrace{\mathtt{APOHGMMW}} & \overbrace{\mathtt{APOHGMMW}} & \overbrace{\mathtt{APOHGMMW}} & ...
    \end{matrix}
 
-At first, this might not seem like a particularly serious problem.
-Assuming the block cipher is secure, it doesn't look like an attacker
-would be able to decrypt anything. By dividing the ciphertext stream up
-into blocks, an attacker would only be able to see that a ciphertext
-block, and therefore a plaintext block, was repeated.
+Initially, this may not seem like a serious problem.
+Assuming the block cipher is secure, it does not appear like an attacker
+can decrypt anything. By dividing the ciphertext stream
+into blocks, an attacker only sees that a ciphertext
+block was repeated, and therefore a plaintext block.
 
-We'll now illustrate the many flaws of :term:`ECB mode` with two attacks. First,
-we'll exploit the fact that repeating plaintext blocks result in
-repeating ciphertext blocks, by visually inspecting an encrypted image.
-Then, we'll demonstrate that attackers can often decrypt messages
+The many flaws of :term:`ECB mode` are illustrated in the following two attack examples. First,
+by visually inspecting an encrypted image, we exploit the fact that repeating 
```

Comment:
This introduces passive voice but also changes the meaning of the sentence, I think, and it makes less sense now. You can exploit the fact that repeating plaintext blocks result in repeating ciphertext blocks _through visual inspection_. But it seems very strange to reverse that and say visual inspection directly you're exploiting anything.

I think this needs rephrasing.